app/components/DocumentListToolbar/DocumentListToolbar.jsx

```diff
@@ -13,6 +13,11 @@ function range(from, to) {
         .map((el, i) => from + i)
 }
 
+// If there are more pages than this, ellipses will be shown.
+const MAX_VISIBLE_PAGES = 9
+// Number of pages around the current one (in each direction) that will be visible.
```

Comment:
Can you add a blank line, pretty please? 😇

Reply:
Done, will commit later